docs/user_guide/layout.rst

```diff
@@ -0,0 +1,449 @@
+For example, to make the content grow to fit all available width, add a custom CSS rule like:
+
+.. code-block:: css
+
+   .bd-content {
+     flex-grow: 1;
+     max-width: 100%;
+   }
+
+Templates and components
+========================
+
+There are a few major theme sections that you can customize to add/remove
+components, or add your own components. Each section is configured with a
+list of *html templates* - these are snippets of HTML that are inserted into
+the section by Sphinx.
+
+You can choose which templates show up in each section, as well as the order in
+which they appear. This page describes the major areas that you can customize.
+
+.. note::
+
+   When configuring templates in each section, you may omit the ``.html``
+   suffix after each template if you wish.
+
+
+.. _layout-header:
+
+Header / Navigation Bar
+=======================
+
+The header is at the top of the page above all other content, and contains site-level information.
+
+
+Header sections
+---------------
+
+The header is broken up into three sections.
+Each section is configured in ``conf.py`` with the following configuration:
+
+- Left section: ``html_theme_options['navbar_start']``
+- Middle menu: ``html_theme_options['navbar_center']``
+- Right section: ``html_theme_options['navbar_end']``
+
+By default, the following configuration is used:
+
+.. code-block:: python
+
+   html_theme_options = {
+   ...
+   "navbar_start": ["navbar-logo"],
+   "navbar_center": ["navbar-nav"],
+   "navbar_end": ["navbar-icon-links"]
+   ...
+   }
+
+Configure the navbar center alignment
+-------------------------------------
+
+By default, the navigation bar center area will align with the content on your
+page. This equals the following default configuration:
+
+.. code-block:: python
+
+   html_theme_options = {
+      ...
+      "navbar_align": "content"
+      ...
+   }
+
+If instead you'd like these items to snap to the left (closer to the logo), use this
+configuration:
+
+.. code-block:: python
+
+   html_theme_options = {
+      ...
+      "navbar_align": "left"
+      ...
+   }
+
+If you'd like these items to snap to the right of the page, use this configuration:
+
+.. code-block:: python
+
+   html_theme_options = {
+      ...
+      "navbar_align": "right"
+      ...
+   }
+
+
+.. _layout-article-header:
+
+Article Header
+==============
+
+The article header is a narrow bar just above the article's content.
+It does not contain anything immediately viewable to the reader, but is kept as a placeholder in case theme developers wish to re-use it in the future.
+
+.. _layout-sidebar-primary:
+
+Primary sidebar (left)
+======================
+
+The primary sidebar is just to the left of a page's main content.
+It is primarily used for between-section navigation.
+By default it will show links to any sublings / children of the current active top-level section (corresponding to links in your header navigation bar).
```

Comment:
```suggestion
By default it will show links to any siblings / children of the current active top-level section (corresponding to links in your header navigation bar).
```